data/tlg0007/tlg088/tlg0007.tlg088.perseus-eng4.xml

```diff
@@ -0,0 +1,554 @@
+<?xml version="1.0" encoding="UTF-8"?> 
+<?xml-model href="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+	<teiHeader xml:lang="eng">
+      <fileDesc>
+         <titleStmt>
+         	<title>Whether the Athenians were more renowned for their warlike achievements or for their learning.</title>
```

Comment:
please capitalize title in keeping with common practice
`Whether the Athenians Were More Renowned For Their Warlike Achievements or For Their Learning`